Doxyfile

```diff
@@ -20,7 +20,7 @@ QUIET                  = YES
 WARN_IF_UNDOCUMENTED   = NO
 
 # TODO: Add the other files. It is just xxhash.h for now.
-FILE_PATTERNS          = xxhash.h
+FILE_PATTERNS          = xxhash.h xxh_x86dispatch.c
```

Comment:
minor : more a question :
why is it `xxh_x86dispatch.c`, rather than `xxh_x86dispatch.h` ?